app/views/sessions/pricing.html.erb

```diff
@@ -1 +1,25 @@
+<br/>
+<img src="/assets/logo.png" alt = "Background">
+<br/>
+<br/>
+
+
 <h1>Pricing</h1>
+#do i need to make this h2 lol
```

Comment:
up to you! we can figure it out next week when we figure out how the website will look across the board.

Reply:
you are the citation and the other code is from https://www.paypal.com/buttons/smart